setup.py

```diff
@@ -75,25 +72,24 @@ def read_file(fname):
         'Development Status :: 5 - Production/Stable',
         'Environment :: Web Environment',
         'Framework :: Django',
-        'Framework :: Django :: 1.11',
         'Framework :: Django :: 2.2',
         'Framework :: Django :: 3.0',
+        'Framework :: Django :: 3.1',
         'Intended Audience :: Developers',
         'License :: OSI Approved :: MIT License',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
         'Topic :: Internet :: WWW/HTTP',
         'Topic :: Software Development :: Libraries :: Python Modules',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.4',
-        'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
-    ],
+        'Programming Language :: Python :: 3.9',
+       ],
     platforms=['any'],
+    python_requires='>=3.6',
+    install_requires=['django>=2.2'],
```

Comment:
Adding this explicitly to the package is a debated topic. This has unwanted side-effects in some occasion (think: when django-analytical is installed before Django is installed).

I would suggest to leave this out.

Reply:
Yes... I've been told off as a releaing packages without this before :shrug:. I'll remove.